src/library_html5.js

```diff
@@ -105,7 +105,11 @@ var LibraryJSEvents = {
         var call = JSEvents.deferredCalls[i];
         JSEvents.deferredCalls.splice(i, 1);
         --i;
+#if MIN_CHROME_VERSION < 46 || MIN_FIREFOX_VERSION < 27 || MIN_IE_VERSION != TARGET_NOT_SUPPORTED || MIN_SAFARI_VERSION < 80000 // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#Spread_in_function_calls
```

Comment:
Wouldn't it make sense to define this centrally?

This brings up the question again in mind of weather we should just switch to ES6 across all our internal code and rely on babel or some other transpiler when shipping to ES5 browsers.   As somebody who ships to a lot of older targets I wonder what you think about this option @juj ?

Reply:
Using babel will be an overkill. There are very few places in code that are ES6 that adding babel machinery to the build will not help these few lines. Also with babel we are no longer in control of what kind of code gets emitted, and looking at what it emits for [emulating spread syntax](https://stackoverflow.com/questions/55130183/how-do-i-detect-fallback-if-the-browser-has-no-spread-syntax-notation-in-fu), it looks pretty bad.